src/Approximations/overapproximate.jl

```diff
@@ -1636,6 +1647,31 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     return Zonotope(c, G)
 end
 
+function _overapproximate_zonotope_cpa(X::AbstractPolytope, ::Type{<:Zonotope},
+                                       dirs=OctDirections)
+    n = dim(X)
+    p = Vector(undef, Int(floor(n/2)))
+    j = 1
+    if isa(X, VPolytope)
+        for i=1:2:n-1
+            p[j] = project(X, [i, i+1])
+            j += 1
+        end
+    else
+        for i=1:2:n-1
+            p[j] = concretize(Projection(X, [i, i+1]))
+            j += 1
+        end
+    end
+
+    if !iseven(n)
+        push!(p, project(X, [n]))
+    end
+
+    Z = [_overapproximate_zonotope_vrep(poly, Zonotope, dirs(2)) for poly in p]
```

Comment:
if `n` is not even, the last block has dimension 1. thn in that case, using `dirs(2)` is wrong?